modules/va_notify/db/migrate/20250123063915_remove_existing_notification_callback_to_field.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveExistingNotificationCallbackToField < ActiveRecord::Migration[7.2]
+  def change
+    safety_assured { remove_column :va_notify_notifications, :to, :string, if_exists: true }
```

Comment:
In the schema, it's :to is not a string but text shown [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/db/schema.rb#L1443)